shark_turbine/kernel/wave/codegen.py

```diff
@@ -372,6 +383,51 @@ def _is_identity_mapping(
     return True
 
 
+def _build_mask(
```

Comment:
Can you add a lit test showing what the IR looks like?

Reply:
Also helps for me to debug and step through locally without having access to the cluster.

Reply:
done

Reply:
thanks!